ui/raidboss/data/07-dt/dungeon/strayborough-deadwalk.txt

```diff
@@ -0,0 +1,197 @@
+# 8EBD Scream -- Unknown. Possibly effects from the pizza slices?
+# 8EF6 Spirited Charge -- Activate noble adds
+# 9B02 Evil Scheme -- Exaflare castbar
+# 9B03 Evil Scheme -- Exaflare floor markers
+# 9B04 Evil Scheme -- Exaflare puddles
+# 9B05 Looming Nightmare -- Chasing AoE castbar
+# 9B06 Looming Nightmare -- Initial chasing AoE
+# 9B07 Looming Nightmare -- Chasing AoE repeated
+
+
+
+#~~~~~~~~~~~~~~~~~#
+# Jack-in-the-pot #
+#~~~~~~~~~~~~~~~~~#
+
+# -ii 8F74 9131
+
+# The Party Plaza will be sealed off
+2000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "13F3" } window 2000,1
+2011.0 "Troubling Teacups" Ability { id: "8F6C", source: "Jack-in-the-Pot" }
+2020.1 "--sync--" Ability { id: "8F71", source: "Stray Phantagenitrix" } # Puppet
+2022.1 "Tea Awhirl" Ability { id: "8F6D", source: "Jack-in-the-Pot" }
+2034.2 "Tricksome Treat" Ability { id: "8F70", source: "Stray Phantagenitrix" }
+2043.0 "Troubling Teacups" Ability { id: "8F6C", source: "Jack-in-the-Pot" }
+2051.1 "--sync--" Ability { id: "8F71", source: "Stray Phantagenitrix" } # Puppet
+2053.1 "Tea Awhirl" Ability { id: "8F6D", source: "Jack-in-the-Pot" }
+2068.2 "Tricksome Treat" Ability { id: "8F70", source: "Stray Phantagenitrix" }
+
+2074.8 label "jackLoop"
+2074.8 "Toiling Teapots" Ability { id: "8F72", source: "Jack-in-the-Pot" }
+2079.5 "Piping Pour 1" Ability { id: "8F73", source: "Spectral Samovar" }
+2084.5 "Piping Pour 2" Ability { id: "8F73", source: "Spectral Samovar" }
+2089.5 "Piping Pour 3" Ability { id: "8F73", source: "Spectral Samovar" }
+2096.9 "Last Drop" Ability { id: "8F76", source: "Jack-in-the-Pot" }
+2106.3 "Troubling Teacups" Ability { id: "8F6C", source: "Jack-in-the-Pot" }
+2114.4 "--sync--" Ability { id: "8F71", source: "Stray Phantagenitrix" } # Puppet
+2116.4 "Tea Awhirl" Ability { id: "8F6D", source: "Jack-in-the-Pot" }
+2131.5 "Tricksome Treat" Ability { id: "8F70", source: "Stray Phantagenitrix" }
+2140.5 "Sordid Steam" Ability { id: "8F75", source: "Jack-in-the-Pot" }
+
+2150.5 "Toiling Teapots" Ability { id: "8F72", source: "Jack-in-the-Pot" } window 30,30 forcejump "jackLoop"
+
+
+# IGNORED ABILITIES
+# 8F74 Mad Tea Party -- Poison teacup expansion
+# 9131 Right Wingblade -- Auto-attack
+
+# ALL ENCOUNTER ABILITIES
+# 8F6C Troubling Teacups -- Summon genies
+# 8F6D Tea Awhirl -- Spinning teacups
+# 8F70 Tricksome Treat -- Ghost explosions after Tea Awhirl
+# 8F71 Puppet -- Ghost testhers to teacups
+# 8F72 Toiling Teapots -- Summon poison teacups
+# 8F73 Piping Pour -- Activate poison teacups
+# 8F74 Mad Tea Party -- Poison teacup expansion
+# 8F75 Sordid Steam -- Raidwide
+# 8F76 Last Drop -- Tank buster
+# 9131 Right Wingblade -- Auto-attack
+
+
+
+#~~~~~~~~~~~#
+# Träumerei #
+#~~~~~~~~~~~#
+
+# -ii 417C 9117 911A 911B
+
+# The Restive Hall will be sealed off
+3000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "13F4" } window 3000,1
+3003.0 "攻撃" Ability { id: "417C", source: "Träumerei" }
```

Comment:
Guessing this is an auto?  Maybe change to `--sync--`?

Reply:
It is, not sure why I left it in there when we aren't really doing that anymore. I just removed it for consistency with the other autos.